frontend/src/Judge/components/ShowCard.js

```diff
@@ -18,39 +19,93 @@ const ButtonContainer = styled.div`
   bottom: 0;
 `
 
-const ShowCard = props => (
-  <Card>
-    <h2>
-      <Link to={`show/${props.id}`}>{props.name}</Link>
-    </h2>
+function renderCardContents (props) {
+  const now = moment()
+  const judgingStart = moment(props.judgingStart)
+  if (now.isBefore(judgingStart)) {
+    return renderBeforeJudging(props)
+  } else {
+    return renderDuringJudging(props)
+  }
+}
+function renderDuringJudging (props) {
+  return (
+    <div>
+      <Col>
+        <dl>
+          <dt>Judging Progress:</dt>
+          <dd>
+            {props.ownVotes.length} / {props.entries.length}
+          </dd>
+          <dt>Judging Ends:</dt>
+          <dd>
+            <Moment format='YYYY/MM/DD'>{props.judgingEnd}</Moment>
+          </dd>
+        </dl>
+      </Col>
+      <Col>
+        <ButtonContainer>
+          <Button
+            size='lg'
+            style={{ cursor: 'pointer' }}
+            tag={Link}
+            to={`show/${props.id}/vote`}
+            // TODO: Conditionally change the text
+          >
+            {props.ownVotes.length === 0 && props.entries.length === 0
+              ? 'Start'
+              : props.ownVotes.length === props.entries.length
+                ? 'Review'
+                : 'Resume'}
+          </Button>
+        </ButtonContainer>
+      </Col>
+    </div>
+  )
+}
+
+function renderBeforeJudging (props) {
+  return (
     <Col>
+      <div> Judging has not started yet. Come back to vote soon! </div>
       <dl>
```

Comment:
I feel like in the future we can be more stylish than a `<dl>` but i'm okay with this for now unless there are large objections

Reply:
Style-wise I think most cards will need a visual overhaul (eg. Admin dashboard), so for MVP-purposes, it's alright. Once we have the MVP and get feedback, we should have finalized the data that should go on all the cards, so it'll be easier to architect the information to make them ＡＥＳＴＨＥＴＩＣ